tests/Integration/Routing/ImplicitModelRouteBindingTest.php

```diff
@@ -40,6 +41,13 @@ protected function defineDatabaseMigrations(): void
             $table->timestamps();
         });
 
+        Schema::create('comments', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+            $table->string('slug');
+            $table->integer('user_id');
+            $table->timestamps();
+        });
+
         $this->beforeApplicationDestroyed(function () {
             Schema::dropIfExists('users');
             Schema::dropIfExists('posts');
```

Comment:
Please add the drop method of the table `comments` in `beforeApplicationDestroyed`